server/build/src/infrastructure/database/postgresql/PostgreSQLTaskRepository.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.PostgreSQLTaskRepository = void 0;
+const task_1 = require("../../../domain/entities/task");
+const PostgreSQLClient_1 = require("./PostgreSQLClient");
+class PostgreSQLTaskRepository {
+    constructor() {
+        this.client = PostgreSQLClient_1.PostgreSQLClient.getInstance();
+    }
+    async getAll() {
+        const result = await this.client.query('SELECT * FROM tasks');
+        return result.map((task) => new task_1.Task(task));
+    }
+    async create(task) {
```

Comment:
You probably should validate the task first before inserting it to the database, right?

Reply:
Yes, you are right, I missed the validation, thank you!

Reply:
Done, I got a mistake here I could not save a task in db, now is fixed.